index.html

```diff
@@ -54,5 +56,250 @@ <h1 class="name">
+        <div class="block1">
+            <img src="images/work4.png" alt="">
+        </div>
+    <!-- details block -->
+        <div class="block2">
+            <!--  -->
+            <h2 class="tonic">Tonic</h2>
+            
+            <div class="tools">
+                <h4 class="canopy">CANOPY</h4>
+                <img src="./images/Counter.png" alt="counter" />
+                <h4 class="counter-shadow">Back End Dev</h4>
+                <img src="./images/Counter.png" alt="counter" />
+                <h4 class="counter-shadow">2015</h4>
+              </div>
+          <!--  -->
+    
+          <div class="project-details">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <!--  -->
+    
+          <div>
+            <ul class="tech-Stack">
+              <li id="html">HTML</li>
+              <li id="css">CSS</li>
+              <li id="js">JAVASCRIPT</li>
+            </ul>
+          </div>
+    
+          <!--  -->
+          <div><button type="button" class="seeProject">see project</button></div>
+    
+        </div>
+    </div>
+</section>
+
+<!-- about me section -->
+<div class="cover"> 
+<section class="about-me">
+
+    <div class="myself">
+        <div class="">
+          <h2 class="headSelf">
+            About Myself
+          </h2>
+
+          <p class="text">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <p class="connect">
+            LET'S CONNECT
+           </p>
+           
+           <div class="social-media">
+               <ul> 
+                   <li><a href="" id="twitter"><img src="images/twitter.png" alt=""></a> </li>
+                   <li><a href="" id="linkin" ><img src="images/Linkedin icon.png" alt=""></a> </li> 
+                   <li><a href="" id="medium"><img src="images/medium.png" alt=""></a> </li>
+                   <li><a href="" id="github"><img src="images/github.png" alt=""></a></li>
+                   <li><a href="" id="angelist"><img src="images/angellist.png" alt=""></a></li>
+               </ul>
+           </div>
+          <p><button type="button" class="about2">Get my resume</button></p>
+        </div>
+<!--  -->
+        <ul class="languages">
+          <li class="">
+            <div class="arrow">
+              <h3 class="lan">Languages</h3>
+              <img class="arrowsign" src="images/Union.png" alt="" />
+            </div>
+
+            <div class="">
+              <img src="/images/Frame 9.png" alt="image" />
+              <img src="/images/Frame 10.png" alt="image" />
+              <img src="/images/Frame 11.png" alt="image" />
+            </div>

```

Comment:
- [ ] [**Required**] As mentioned by my colleague before, you must use `ul` and `li` tags for lists of related elements, to fix it, you could update your `div` with a `ul` element and wrap your `img` inside `li` elements